src/sestest/ServerOptions.cs

```diff
@@ -36,6 +36,11 @@ public class ServerOptions
         public string Audience { get; }
 
         /// <summary>
+        /// The expected issuer of the tokens we will verifiy
```

Comment:
Perhaps something like "The expected issuer against which the server verifies tokens." or "The issuer which the server should expect to see when verifying tokens."?